Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs

```diff
@@ -444,10 +448,22 @@ private SingleFileTransferResult NoSourceOperation(BatchContext context, Source
             case SourceAction.Ignore:
                 return new SingleFileTransferResult { Success = true, ActionSkipped = true, ErrorMessages = { msg } };
             default:
-                throw new Exception("Unknown operation in NoSourceOperation");
+                throw new Exception("Unknown operation in NoSourceOperation.");
         }
     }
 
+    /// <summary>
+    /// Handles logging of actions.
```

Comment:
Do we need a comment here? If yes, then do expand it a bit, because the current comment does not add much to clarity. Or otherwise maybe remove this?..